Makefile-docker

```diff
@@ -28,7 +28,10 @@ check_debian_packages: ## check the existence of multiple debian packages
 
 .PHONY: check_pip_packages
 check_pip_packages: ## check the existence of multiple python packages
-	./scripts/check_pip_packages.sh prod.txt dev.txt
+	@ ./scripts/check_pip_packages.sh prod.txt
+	@if [ "$(DOCKER_TARGET)" = "development" ]; then \
```

Comment:
I don't think it will help much. WE have loads of examples of this "./manage.py" is a magic keyword.

Ultimately the source is the dockerfile and there isn't really a way to "parameterize" that afaik.

Reply:
I looked in Dockerfile and it wasn't clear to me what the possible values of `DOCKER_TARGET` were, and what behaviour defining them has.  Is this already documented?  (What happens if `DOCKER_TARGET` is set to something _other_ than "production" or "development"?)

Reply:
Each stage `from <name> as <stage>` can be targeted in a docker build. It probably could be better doucmented but I would put that in a separate PR. https://mozilla.github.io/addons-server/topics/development/building_and_running_services.html

You can set any value and if you pass a value that isn't an actual stage it will probably error. But we actually only care about the binary choice of "production" or not production. Anything not production is DEV_MODE.

Reply:
🤔  interesting - so should this `@if ...` be testing `<> "production"` (or whatever the bash syntax is)?  (one less magic keyword )

Reply:
Well if you look at what the condition is doing, we actually care here that the target is "development" that is the stage where we expect dev.txt to be installed.. only there. the != production check is not good enough, in this particular case.

Reply:
Done